### PR TITLE
zh-translation: docs/setup/platform-setup/azure/index.md

### PR DESCRIPTION
**Pull request by TianTianBigWang**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/setup/platform-setup/azure/index.md [hunk 1]
```diff
@@ -82,12 +82,12 @@ test: no
     {{< /text >}}
 
     {{< tip >}}
-    几分钟之后，你能发现你的集群在你的 Azure subscription 上的
-    resource group 里被叫做 `<dns_prefix>-<id>`。假设 `dns_prefix` 有这样的值 `myclustername`，一个带着唯一集群 ID `mycluster-5adfba82` 的有效的 resource group。`aks-engine` 在 `_output` 文件夹中生成你的 `kubeconfig`
+    几分钟之后，你能在名为 `<dns_prefix>-<id>`的 Azure subscription 的资源组中找到你的集群。
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/azure/index.md:85]:
```suggestion
    几分钟之后，您能在名为 `<dns_prefix>-<id>`的 Azure subscription 的资源组中找到你的集群。
```

File: content/zh/docs/setup/platform-setup/azure/index.md [hunk 2]
```diff
@@ -82,12 +82,12 @@ test: no
...
+    假设 `dns_prefix` 有这样的值 `myclustername`，一个有效的资源组具有唯一集群 ID 为 `mycluster-5adfba82` 。`aks-engine` 在 `_output` 文件夹中生成你的 `kubeconfig`
     文件。
     {{< /tip >}}
 
-1. 使用 `<dns_prefix>-<id>` 集群 ID，为了从 `_output` 文件夹复制你的 `kubeconfig` 到你的机器：
+1. 使用 `<dns_prefix>-<id>` 集群 ID，将 `kubeconfig` 从 `_output` 文件夹复制到你的机器：
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/azure/index.md:90]:
```suggestion
1. 使用 `<dns_prefix>-<id>` 集群 ID，将 `kubeconfig` 从 `_output` 文件夹复制到您的机器：
```

File: content/zh/docs/setup/platform-setup/azure/index.md [hunk 3]
```diff
@@ -64,10 +64,10 @@ test: no
     $ wget https://raw.githubusercontent.com/Azure/aks-engine/master/examples/service-mesh/istio.json
     {{< /text >}}
 
-    注意：可能使用其他可以和 Istio 一起工作的 api 模型定义。MutatingAdmissionWebhook 和 ValidatingAdmissionWebhook 准入控制标识和 RBAC 被默认打开。从 [aks-engine api 模型默认值](https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md)获取更多信息。
+    注意：可以使用其他将与 Istio 一起工作的 api 模型定义。默认情况下，MutatingAdmissionWebhook 和 ValidatingAdmissionWebhook 准入控制标识和 RBAC 会被启用。参阅 [aks-engine api 模型默认值](https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md)获取更多信息。
 
-1. 使用 `istio.json` 模板来部署你的集群。你能发现对于参数的参考在
-   [官方文档](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/deploy.md#step-3-edit-your-cluster-definition)中。
+1. 使用 `istio.json` 模板来部署你的集群。你能在
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/azure/index.md:69]: 您。

File: content/zh/docs/setup/platform-setup/azure/index.md [hunk 4]
```diff
@@ -64,10 +64,10 @@ test: no
...
+   [官方文档](https://github.com/Azure/aks-engine/blob/master/docs/topics/creating_new_clusters.md#deploy)中找到对参数的参考。
```
Comment (kebe7jun) [inline content/zh/docs/setup/platform-setup/azure/index.md:70]: 不要换行。

**Review by kebe7jun — APPROVED** [review]
lgtm
/retest

**Pull request merged**